eng/pipelines/runtime-staging.yml

```diff
@@ -62,44 +62,44 @@ extends:
       - ${{ if eq(variables.dependOnEvaluatePaths, true) }}:
         - template: /eng/pipelines/common/evaluate-default-paths.yml
 
-      #
-      # Build the whole product using Mono and run runtime tests with the JIT.
-      #
-      - template: /eng/pipelines/common/platform-matrix.yml
-        parameters:
-          jobTemplate: /eng/pipelines/common/global-build-job.yml
-          helixQueuesTemplate: /eng/pipelines/coreclr/templates/helix-queues-setup.yml
-          buildConfig: Release
-          runtimeFlavor: mono
-          platforms:
-            - iossimulator_x64
-          variables:
-            - ${{ if and(eq(variables['System.TeamProject'], 'public'), eq(variables['Build.Reason'], 'PullRequest')) }}:
-              - name: _HelixSource
-                value: pr/dotnet/runtime/$(Build.SourceBranch)
-            - ${{ if and(eq(variables['System.TeamProject'], 'public'), ne(variables['Build.Reason'], 'PullRequest')) }}:
-              - name: _HelixSource
-                value: ci/dotnet/runtime/$(Build.SourceBranch)
-            - name: timeoutPerTestInMinutes
-              value: 60
-            - name: timeoutPerTestCollectionInMinutes
-              value: 180
-          jobParameters:
-            testGroup: innerloop
-            nameSuffix: AllSubsets_Mono_RuntimeTests
-            runtimeVariant: minijit
-            buildArgs: -s mono+libs -c $(_BuildConfig)
-            timeoutInMinutes: 240
-            condition: >-
-              or(
-                eq(dependencies.evaluate_paths.outputs['SetPathVars_runtimetests.containsChange'], true),
-                eq(dependencies.evaluate_paths.outputs['SetPathVars_mono_excluding_wasm.containsChange'], true),
-                eq(variables['isRollingBuild'], true))
-            ${{ if eq(variables['isRollingBuild'], true) }}:
-              # extra steps, run tests
-              extraStepsTemplate: /eng/pipelines/common/templates/runtimes/build-runtime-tests-and-send-to-helix.yml
-              extraStepsParameters:
-                creator: dotnet-bot
-                testRunNamePrefixSuffix: Mono_$(_BuildConfig)
-              extraVariablesTemplates:
-                - template: /eng/pipelines/common/templates/runtimes/test-variables.yml
+      # #
+      # # Build the whole product using Mono and run runtime tests with the JIT.
```

Comment:
Is this temporary? If it's not, is this healthy enough to move out of runtime-staging and delete the yml?

Reply:
Yes, all jobs that are not used for testing and validation have been temporarily disabled. They will be enabled before the code review and merge.